src/js/components/Paragraph.js

```diff
@@ -31,5 +32,6 @@ Paragraph.propTypes = {
   align: PropTypes.oneOf(['start', 'center', 'end']),
   id: PropTypes.string,
   margin: PropTypes.oneOf(['none', 'small', 'medium', 'large']),
-  size: PropTypes.oneOf(['small', 'medium', 'large'])
+  size: PropTypes.oneOf(['small', 'medium', 'large']),
+  width: PropTypes.oneOf(['small', 'medium', 'large'])
```

Comment:
@ericsoderberghp I've submitted a separate PR for adding sizing to Box.  I believe this should work as desired, so please review it when you get a chance.  Thanks!